app/src/main/res/xml/settings.xml

```diff
@@ -24,6 +24,14 @@
         android:defaultValue="@string/settings_dark_default"
         android:key="@string/settings_dark_key"
         android:title="@string/settings_dark_label"/>
+    <EditTextPreference
+        android:defaultValue="0"
+        android:inputType="number"
+        android:key="@string/settings_autorewind_key"
+        android:selectAllOnFocus="true"
+        android:singleLine="true"
+        android:maxLength="1"
```

Comment:
Better set maxLength to 2. I think it is plausible that some people would like to set the autorewind time to more than 9 seconds.